lib/danger/danger_core/plugins/dangerfile_bitbucket_server_plugin.rb

```diff
@@ -0,0 +1,36 @@
+# coding: utf-8
+require 'danger/plugin_support/plugin'
+
+module Danger
+  class DangerfileBitbucketServerPlugin < Plugin
+    def self.new(dangerfile)
+      return nil if dangerfile.env.request_source.class != Danger::RequestSources::BitbucketServer
+      super
+    end
+
+    def self.instance_name
+      "bitbucket_server"
```

Comment:
There is actually a difference between Bitbucket and Bitbucket Server. Bitbucket Server is just the new name for Stash. Bitbucket is the Github-like service. So it's like Github and Github Enterprise.